sdk/storage/Azure.Storage.Blobs/tests/BlobSasBuilderTests.cs

```diff
@@ -38,6 +40,19 @@ private static UserDelegationKey GetUserDelegationKey(TestConstants constants)
                 Value = constants.Sas.KeyValue
             };
 
+        private async Task<BlobBaseClient> GetNewBlobClient(BlobContainerClient container, string blobName = default)
```

Comment:
Should this be called GetBlobWithContent or CreateBlob or so ? Also consider moving this to BlobTestBase, we have other places where we do something like that.